i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx

```diff
@@ -0,0 +1,215 @@
+---
+sidebar_position: 2
+---
+
+# Setup & Configuration
+
+## Configuration
+
+### A note on individual tastes
+
+::::caution
+evcc has a lot of settings, knobs, and dials. The default parameters are based on lots of experience from many users, and most of the time, the default should work well for you.
+
+**If you do start making changes, remember the golden rule:**
+
+:::danger
+**Since parameters sometimes influence each other, it is STRONGLY encouraged to only make changes to one setting at at time, and test the effects of each change before making further modifications.**
+:::
+::::
+
+### Syntax Error in evcc.yaml
+
+Yaml is very sensitive to syntax - and errors don't always catch the eye straight away. Linters such as (https://onlineyamltools.com/validate-yaml) can be super useful, and are worth checking to find simple mistakes.
+
+## Common Problems
+
+### Something's not working. What now?
+
+We have a [Community Support](https://github.com/evcc-io/evcc/discussions) forum, with developers and users standing by to help solve common issues.
+
+When making a post, it would be super useful to give as much information as possible on the following:
+
+- As precise an explanation as possible as to what your problem is
+
+- Which devices (vehicles, meters, chargers) are in use?
+
+- Your full, sanitised `evcc.yaml`
+
+<details>
+<summary>
+   Helper comamnd (*please make sure all usernames, passwords, and other secrets are removed!*).
+</summary>
+
+`egrep -v '^#|^\s+#' evcc.yaml |grep . |sed -E 's/(password|user|token|url|host|broker).*/\1: *****/'`
+
+</details>
+
+  - Alternatively, you can use `evcc dump --cfg >~/config.txt` to create an anonymized configuration file at `~/config.txt`.
+
+
+**⛑️ First Aid**
+
+We've added a useful helper utility to evcc to help with the above. By entering `evcc discuss` in your terminal, our helper wizard will help you create a new discussion topic in our "First Aid" section.
```

Comment:
The topic in GitHub discussions is not localized and GH gives us no option to add localization. So we should keep Erste Hilfe" when describing the navigation. At least for now. Maybe add a notice that it's totally fine to also post in English.

Renaming the GH topic to English is something we should do eventually.

```suggestion
We've added a useful helper utility to evcc to help with the above. By entering `evcc discuss` in your terminal, our helper wizard will help you create a new discussion topic in our "Erste Hilfe" section.
```

Reply:
Concur, should we make this read `"Erste Hilfe" (first aid)` to make it nice and clear what the section means?

Reply:
Good point. I changed it.